api/src/certification/complementary-certification/application/attach-target-profile-route.js

```diff
@@ -0,0 +1,67 @@
+import { attachTargetProfileController } from './attach-target-profile-controller.js';
+import { securityPreHandlers } from '../../../../lib/application/security-pre-handlers.js';
+import Joi from 'joi';
+import { identifiersType } from '../../../../lib/domain/types/identifiers-type.js';
+
+const register = async function (server) {
+  server.route([
+    {
+      method: 'PUT',
+      path: '/api/admin/complementary-certifications/{complementaryCertificationId}/badges',
+      config: {
+        pre: [
+          {
+            method: (request, h) =>
+              securityPreHandlers.adminMemberHasAtLeastOneAccessOf([
+                securityPreHandlers.checkAdminMemberHasRoleSuperAdmin,
+                securityPreHandlers.checkAdminMemberHasRoleSupport,
+                securityPreHandlers.checkAdminMemberHasRoleMetier,
+              ])(request, h),
+            assign: 'hasAuthorizationToAccessAdminScope',
+          },
+        ],
+        validate: {
+          params: Joi.object({
```

Comment:
Afin de bien documenter notre swagger (et outils qui se basent dessus), surtout dans notre cas où on renvoie une HTTP 204 plutôt qu'une HTTP 201.
Il y a quelques exemples dans les autres API

Ci-dessou (l'indentation est pourrie par contre)

```suggestion
        validate: {
          response: {
          failAction: 'ignore',
          status: {
            204: Joi.string().empty(''),
            400: erreurDoc,
            401: erreurDoc,
            403: erreurDoc,
          },
        },
        plugins: {
          'hapi-swagger': {
            produces: ['application/json'],
          },
        },
          params: Joi.object({
```

En résultat ça donne quand même quelque chose de bcp plus propre :


ACTUEL

![image](https://github.com/1024pix/pix/assets/170271/2afe4b53-c740-4b6e-85cb-f4d560a6ecd8)

AVEC LA SUGGESTION ça donnera un truc comme (ici un exemple de 200)

![image](https://github.com/1024pix/pix/assets/170271/98318bc3-bb3c-47de-8999-2ac6fe32e007)